### PR TITLE
fix: proper styles.css build

### PR DESCRIPTION
**Pull request by suevalov**
# Purpose of PR

Fix missing CSS variables

Fixes https://github.com/contentful/forma-36/issues/663

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/tools/postcss.config.js [hunk 1]
```diff
@@ -11,8 +11,11 @@ module.exports = {
     postCssUrl,
     postcssPresetEnv({
       stage: 0, // Allow experimental CSS features
-      browsers: 'last 2 versions, ie >= 11',
+      browsers: ['Chrome >= 75', 'Edge >= 17', 'Firefox >= 70', 'Safari >= 12'],
       preserve: false, // Remove the pre-polyfilled CSS
     }),
+    // require('cssnano')({
```
Comment (m10l) [inline packages/forma-36-react-components/tools/postcss.config.js:17]: I guess we can remove this?

File: packages/forma-36-react-components/package.json [hunk 2]
```diff
@@ -93,6 +93,7 @@
     "postcss-import": "^12.0.1",
     "postcss-preset-env": "^6.7.0",
     "postcss-url": "^8.0.0",
+    "cssnano": "^4.1.10",
```
Comment (m10l) [inline packages/forma-36-react-components/package.json:96]: And we can remove this?

**Review by m10l — APPROVED** [review]
@suevalov Whilst looking into this issue I noticed that if I remove uses of `compose` (like `composes: focus-outline--default from './../../styles/settings/a11y.css';`) the CSS is built correctly (variables are removed from the built and replaced with the hardcoded values instead as expected).

Happy to accept that PR just to get shit moving again, but it might not really be solving the root cause.

File: packages/forma-36-react-components/tsdx.config.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ module.exports = {
         config: {
           path: path.resolve(__dirname, './tools/postcss.config.js'),
         },
+        minimize: true,
```
Comment (m10l) [inline packages/forma-36-react-components/tsdx.config.js:16]: 🙇

**Comment by denkristoffer** [comment]
Would be cool if we could synchronise browser targets automatically in the future. Is that even possible? 😄

**Comment by suevalov** [comment]
@denkristoffer https://github.com/browserslist/browserslist#shareable-configs

Feel free to create one and rely on it in our repos

**Pull request merged**